controllers/notifications.js

```diff
@@ -0,0 +1,41 @@
+const notificationQuery = require('../models/notifications')
+/**
+ * Fetches the data about our users
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+module.exports.getNotificationsForUser = async (req, res) => {
+  try {
+    const { page: currentPage, n: limit } = req.query
+    const { id: userId } = req.userData
+
+    if (!currentPage || !limit) {
+      logger.error('currentPage/ limit not present in the query')
+      return res.boom.badRequest('Please check your query again')
+    }
```

Comment:
Query params are optional
We need not do these checks. Instead we should have use some default value if query params are not externally defined.

Reply:
Is it fine If I set currentPage as 1 and the limit as 10?

Reply:
Yes, that works 👍🏻